README.md

```diff
@@ -13,9 +13,8 @@ Generate JUnit compatible XML reports in Rust.
 
 ```rust
 
-    extern crate junit_report;
-
-    use junit_report::{ReportBuilder, TestCase, TestCaseBuilder, TestSuite, TestSuiteBuilder, Duration, TimeZone, Utc};
+    use junit_report::{ReportBuilder, TestCase, TestCaseBuilder, TestSuite, TestSuiteBuilder};
+    use time::{macros::datetime, Duration};
```

Comment:
Why switch to `use time::..` instead of re-exporting them from `junit_report` as it was done for `Duration, TimeZone, Utc` before?

Reply:
@bachp I don't consider re-exporting other crates (especially basic once like `time`) a good practice for couple of reasons
1. This may confuse `IDE` on where to import those structs from. For example in case you have `time` as a local dependency or some other crate re-exports it too. I think of it as "extended std" and like I wouldn't re-export `std` stuff, same applies here. 
  Good example here would be `futures` and `tokio` relation. `tokio` depends on `futures`, but doesn't re-export it, exactly because `futures` is de-facto standard  in the rust ecosystem. 
2. If `time` `0.4` will be released, running locally `cargo outdated` would produce error and someone eventually will open issue about it or fix it themself. On the other hand having it hidden inside crate makes it a bit harder to keep track of what is outdated and what isn't. 

But I saw libraries going both direction and I don't think we as a rust community have consensus on this topic. So if you think it's better not to re-export, I'll fix it.

Reply:
As I understand re-exporting was considered a good practice when `extern crate` was a thing. [But now opinions are split](https://github.com/rust-lang/api-guidelines/discussions/176)

Reply:
@ilslv it's still neat to re-export the types which are intended to be frequently used while using the crate. This allows to simply add the crate as a dependency and use it without a need to lurk which crates should be added more (Cargo doesn't have a notion of peer dependencies like NPM does). It's good for a crate to be ready for use out-of-the-box.

Regarding `cargo outdated`, I think the situation is opposite to what you've described:  
Imagine a situation where you need `time` only because you're using `junit-report`. Then a `0.4` version of `time` is released, and now you're in a weird state with `cargo outdated`: it complains about usage of `time` `0.3`, but you cannot actually bump it up until `junit-report` does so. It would be OK if you've opted for `time` not only because of `junit-report` (you still may use different versions in different part of your code), but it's quite weird to end up with dependecy problems of third-party crate in your workspace. If you don't opt for `time` directly because of `junit-report`, all those problems are solved transparately for your workspace.

So I agree with @bachp about preserving re-exports.

Reply:
Thanks for the explanations.

May main reason for re-exporting was exactly that the users don't need to care about `chrone` or `time` they just use `junit-report`.

@ilslv Can you change the PR accordingly.